### PR TITLE
Display Tweaks

### PR DESCRIPTION
**Pull request by kilikfan**
Had a play around with some display tweaks. And fixed the Game End screen

### CODE REVIEW

**Comment by kilikfan** [comment]
I still can't quite get the Play Page right, the hand never takes up all the room it has allotted to it [like this](https://paste.pics/BSG7I)

**Comment by kilikfan** [comment]
I worked out how the hand works! Previously it was anchored to the bottom of the frame due to the alignments used. I've done one of those cursed one-liner changes that puts it more into the middle of its box. I welcome comments.

**Comment by ghost** [comment]
image

There seems to be a large-ish gap now between the card heading and the actions. Even on a large device, the actions are no longer fitting entirely on the card without scrolling.

**Comment by ghost** [comment]
These could probably do with being a bit narrower in height too
image

**Review by ghost — APPROVED** [review]
Happy for you to merge this and for us to continue to iterate small improvements on the layouts. I might take a look at a few bits when i next have some time!

**Comment by kilikfan** [comment]
Couple of pics of the newest tweaks: [Big phone](https://paste.pics/BT90U) [Small phone](https://paste.pics/BT912)

File: lib/screens/play/card/card_header.dart [hunk 1]
```diff
@@ -11,17 +11,28 @@ class CardHeader extends StatelessWidget {
...
-        ]));
+        padding: const EdgeInsets.all(10),
+        child: Column(children: _getHeadings(headerText, subHeaderText)));
+  }
+
+  List<Widget> _getHeadings(String headerText, String subHeaderText) {
+    List<Widget> headings;
+
+    //Card heading should always be displayed
+    headings = [heading(headerText)];
```
Comment (ghost) [inline lib/screens/play/card/card_header.dart]: Could combine these
```suggestion
    //Card heading should always be displayed
    final headings = [heading(headerText)];
```

File: lib/screens/play/player_hand.dart [hunk 2]
```diff
@@ -11,27 +11,41 @@ class PlayerHand extends StatelessWidget {
...
+              ));
+        })
+        .values
+        .toList();
+
+    Widget _resourceCount() {
+      return Align(
+          alignment: AlignmentDirectional.topEnd, // Top right corner
+          child: Text("Count: ${hand.cards.length}"));
+    }
```
Comment (ghost) [inline lib/screens/play/player_hand.dart]: Doesn't really make sense for this to be a nested method?
```suggestion
    final resourceCount = Align(
          alignment: AlignmentDirectional.topEnd, // Top right corner
          child: Text("Count: ${hand.cards.length}"));
    }
```

**Review by ghost — APPROVED** [review]
Nice, bags of room! :rocket:

**Pull request merged**